hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientWithRatis.java

```diff
@@ -323,7 +323,8 @@ public void testParallelDeleteBucketAndCreateKey() throws IOException,
     omSM.getHandler().setInjector(injector);
     thread1.start();
     thread2.start();
-    Thread.sleep(2000);
+    // Wait long enough for createKey's preExecute to finish executing
+    Thread.sleep(10000);
```

Comment:
Can we change this to `waitFor` some explicit condition?

Reply:
Yes, but this needs to modify more codes. We need to know the `preExecture` has been executed completely, but the  `preExecture` is a `prviate` method (or we need to add some flag to watch), I don't want ot modify it just for only one testing. 
Add sleeping time is a dirty but quick solution. How do you think?